configurable_lti_consumer/templates/html/student.html

```diff
@@ -68,7 +63,7 @@ <h2 class="problem-header">
         </section>
     % endif
     % if launch_target == 'iframe':
-        <div style="height:${inline_height}px;">
+        <div>
```

Comment:
I think we can't just remove it, we need to make it configurable via our lti options so that it can be set to auto-resize for some LTI services only.

Reply:
activate it when the field is null sounds good.